src/targets/browser/browserLauncher.ts

```diff
@@ -98,17 +98,25 @@ export abstract class BrowserLauncher<T extends AnyChromiumLaunchConfiguration>
     // dir unless it's  explicitly requested.
     let resolvedDataDir: string | undefined;
     if (typeof userDataDir === 'string') {
-      resolvedDataDir = userDataDir;
+      resolvedDataDir = path.resolve(userDataDir);
     } else if (userDataDir) {
-      resolvedDataDir = path.join(
-        this.storagePath,
-        runtimeArgs?.includes('--headless') ? '.headless-profile' : '.profile',
+      resolvedDataDir = path.resolve(
+        path.join(
+          this.storagePath,
+          runtimeArgs?.includes('--headless') ? '.headless-profile' : '.profile',
+        ),
       );
     }
 
-    try {
-      fs.mkdirSync(this.storagePath);
-    } catch (e) {}
+    if (!fs.existsSync(this.storagePath)) {
```

Comment:
The `if` checks should not be necessary for recursive `mkdirSync`

Reply:
Oh, yes, I have tested and confirmed that error EEXIST can be thrown by mkdirSync only when non-recursive, so existsSync is not necessary.
That commit has been amended.